tests/integration/mininet_tests.py

```diff
@@ -154,13 +154,18 @@ def verify_events_log(self, event_log, timeout=10):
 
     def test_untagged(self):
         """All hosts on the same untagged VLAN should have connectivity."""
-        self._enable_event_log(10)
+        self._enable_event_log()
         self.ping_all_when_learned()
         self.flap_all_switch_ports()
         self.verify_traveling_dhcp_mac()
         self.gauge_smoke_test()
         self.prometheus_smoke_test()
         self.assertGreater(os.path.getsize(self.event_log), 0)
+        controller = self._get_controller()
```

Comment:
Don't understand this bit... whazzup?   If there's a reason maybe add a comment because it looks like it's a no-op because the event socket stuff isn't written anywhere... ?